### PR TITLE
Stress testing

### PR DESCRIPTION
**Pull request by wizard-jS**

### CODE REVIEW

File: client/app.jsx [hunk 1]
```diff
@@ -89,28 +89,32 @@ export default class CommentSection extends React.Component {
...
-      this.state.displayedComments.map((i, index) => (
+      this.state.displayedComments.map((i, index) => i && (
         <Comment 
+
         key={index}
-        image={i.profilePic}
+        image={i.profilepic}
         username={i.username}
-        songTime={i.songTime}
+        songTime={i.songime}
```
Comment (Ceciil) [inline client/app.jsx]: songtime*

File: server/model.js [hunk 2]
```diff
@@ -23,16 +26,17 @@ module.exports = {
...
     const {
-      songId, profilePic, username, message, postedAt, songTime, followers,
+      songid, stringid, profilepic, username, message, postedat, songtime, followers
     } = params;
-    console.log('these are the params', params);
-    const createCommentQuery = `INSERT INTO comments (songId, profilePic, username, message, postedAt, songTime, followers)
-                                VALUES  ( "${songId}","${profilePic}","${username}","${message}","${postedAt}","${songTime}","${followers}")`;
-    connection.query(createCommentQuery, (err, newComment) => {
+    const createCommentQuery = `INSERT INTO comments (songid,profilePic,username,message,postedat,songtime,followers)
+                                VALUES  (${songid},'${profilepic}','${username}','${message}','${postedat}','${songtime}',${followers})`;
```
Comment (Ceciil) [inline server/model.js]: Will you be inserting stringid as well?

File: server/controller.js [hunk 3]
```diff
@@ -30,25 +19,14 @@ module.exports = {
...
-    model.fetchNumberOfCommentsWithStringId(stringId, (err, data) => {
-      if (err) {
-        console.log('Error fetching number of comments with string id: ', err);
-        res.sendStatus(404);
-      } else {
-        res.send({ count: data[0]['COUNG(*)']});
-      }
+        res.status(200).send(data.rows[0].count);
+        // res.send({ count: data[0]['COUNT(*)'] });
+      }app.use(bodyParser.json());
```
Comment (Ceciil) [inline server/controller.js]: Formatting issue?

File: server/model.js (hunk 2 above)
Comment (wizard-jS) [reply server/model.js]: No, stringId as been removed after bench marking whether a string song field was faster that a number song filed. The answer is, it doesn't matter as long as the field is indexed.

File: server/controller.js (hunk 3 above)
Comment (wizard-jS) [reply server/controller.js]: Thanks. I will check this out.

File: server/model.js [hunk 4]
```diff
@@ -1,19 +1,21 @@
 /* eslint-disable linebreak-style */
-const mysql = require('mysql');
-const config = require('../configDB.js')
-const connection = mysql.createConnection(config)
 
+const pg = require('pg');
+const config = require('../configDB.js');      
+const client = new pg.Client(config);
+
+client.connect();
```
Comment (Ceciil) [inline server/model.js]: You might want to add an error handler, just in case your connection doesn't go through!

**Pull request merged**